packages/permission-log-controller/src/helpers.ts

```diff
@@ -0,0 +1,21 @@
+// helper functions
+
+/**
+ * Get a map from account addresses to the given time.
+ *
+ * @param accounts - An array of addresses.
+ * @param time - A time, e.g. Date.now().
+ * @returns A string:number map of addresses to time.
+ */
+export function getAccountToTimeMap(
```

Comment:
Since this function is now extracted to its own file, I'm worried that someone looking at this might think that it is untested when it's not. Can you add tests for this function? Alternately, if you are confident that this is doing what it's supposed to be doing, you could move it back to PermissionLogController.